### ISSUE TITLE
 Warlock Spell Buged: Demonic Ritual,Inferno,Cruse off Doom,Enslave Deamon

### ISSUE THREAD

**Issue by soso89** [Atlantiss/BugTracker#10288]
Hello
1. ritual of doom - it happens that the demon does not appear at all after a successful procedure for the game
     - The evoked demon is losing very fast, just in a minute, even if I use it to enslave demon
        so after half a minute it disappears what should not.
-Go guard me bugly spel that rain of fire, after all, I couldn't try because it disappears.

2. Inferno - it's exactly like the ritual of doom after the daemon gets lost very fast and also is immue on enslave deamon.

3.cruse of doom - this does not even work I wrote it and gm but he told me that it has a small chance but I tried it 100x and nothing ever happened, so to say that it has a small chance of 100x is probably stupid.

4. Enslave Deamon - This spel also doesn't work exactly on someone's acting on someone but maybe it's not even a bug in spell as in the daemon settings themselves.

In conclusion, I would like to say they are very spelly used so it takes from the reserve of can have some bugs, but in any case it would be good to take a look and put it in order to make it as it was to me. I apologize in advance for my English.
